packages/wrangler/src/pages/dev.tsx

```diff
@@ -431,3 +439,15 @@ async function spawnProxyProcess({
 
 	return port;
 }
+
+const blockWorkerJsImports: Plugin = {
+	name: "block-worker-js-imports",
+	setup(build) {
+		build.onResolve({ filter: /.*/g }, (_args) => {
+			logger.error(
+				`_worker.js is using an import. This will throw an error in production.\nYou should bundle your worker before deploying it, or instead remove your import.`
```

Comment:
```suggestion
				`_worker.js is importing from another file. This will throw an error if deployed.\nYou should bundle your Worker or remove the import if it is unused.`
```